ckcp/gitops.sh

```diff
@@ -135,12 +141,12 @@ install_openshift_gitops() {
   #############################################################################
 
   # Allow ArgoCD to control resources outside of its namespace
-  oc adm policy add-cluster-role-to-user cluster-admin -z openshift-gitops-argocd-application-controller -n openshift-gitops >/dev/null
+  # oc adm policy add-cluster-role-to-user cluster-admin -z openshift-gitops-argocd-application-controller -n openshift-gitops >/dev/null
 
   # Log into ArgoCD
   echo -n "  - ArgoCD Login: "
   local argocd_password
-  argocd_password="$(oc get secret openshift-gitops-cluster -n openshift-gitops -o jsonpath="{.data.admin\.password}" | base64 --decode)"
+  argocd_password="$(kubectl get secret openshift-gitops-cluster -n openshift-gitops -o jsonpath="{.data.admin\.password}" | base64 --decode)"
   argocd login "$ARGOCD_HOSTNAME" --grpc-web --insecure --username admin --password "$argocd_password" >/dev/null
```

Comment:
If it matters: openshift-gitops namespace and openshift-gitops-cluster are OpenShift specific

Reply:
I removed all of Kubernetes specific, for now this script just support Openshift. I will add Kubernetes support in a separate script.

Reply:
could you please then rename the script so that it reflects that? something like openshift_dev_setup.sh

Reply:
@fgiloux sure , the script was renamed to `./ckcp/openshift_dev_setup.sh`